test/fixtures/rules/std_rule_cases/L025.yml

```diff
@@ -65,10 +65,6 @@ test_fail_table_alias_not_referenced_2_subquery:
   fail_str: SELECT * FROM (SELECT * FROM my_tbl foo)
   fix_str: SELECT * FROM (SELECT * FROM my_tbl)
 
-test_fail_subquery_alias_not_referenced:
```

Comment:
Could we leave this in (as a pass now) so we have an implicit and explicit values table test?

Reply:
Sure!